crates/binstalk-fetchers/src/gh_crate_meta.rs

```diff
@@ -41,7 +42,7 @@ impl GhCrateMeta {
                 repo,
```

Comment:
You can return these information in the future at the end of the function, then you can use this to render signature file.